tests/unit-test-b-velverlet/unit-test-b-velverlet.h

```diff
@@ -0,0 +1,43 @@
+/* physical constant and inputs*/
+const double NATOMS = 3;
+const double MASS = 39.948;
+const double EPSILON = 0.2379;
+const double SIGMA = 3.405;
+const double RCUT = 8.5;
+const double BOX = 8.5;
+const double NSTEPS = 1;
+const double DT = 5.0;
+const double VX = 1.0;
+const double VY = 1.0;
+const double VZ = 1.0;
+const double FX = 0.0;
+const double FY = 0.0;
+const double FZ = 0.0;
+const double PX = 0.0;
+const double PY = 0.0;
+const double PZ = 0.0;
+const double PXX = 4.0;
+const double PYY = 4.0;
+const double PZZ = 4.0;
+const double PXXX = 5.0;
+const double PYYY = 5.0;
+const double PZZZ = 5.0;
+const double P0ex = 5.0;
```

Comment:
You can write something like:

`const double Pex[] = {5.0, 9.0, 10.0}`

This will allow you to use: `Pex[0], Pex[1], ..` for your loop